eclass/bitcoincore.eclass

```diff
@@ -12,7 +12,7 @@
 #
 # The eclass provides all common dependencies as well as common use flags.
 
-has "${EAPI:-0}" 5 || die "EAPI=${EAPI} not supported"
+has "${EAPI:-0}" 5 6 || die "EAPI=${EAPI} not supported"
```

Comment:
`epatch` is only deprecated, not forbidden, by EAPI 6. Until EAPI 5 support is dropped, it would just be clutter to try to switch between the two...